src/postload.lua

```diff
@@ -1,3 +1,10 @@
+-- This script is executed only on the first load of the module.
+--
+-- It is NOT executed after hot reload (when the package.loaded
+-- entry is removed and require is called once again).
```

Comment:
It seems, you forgot to adjust this comment.

Reply:
Yes, thanks. Fixed.